packages/expo-updates/ios/EXUpdates/EXUpdatesDevLauncherController.m

```diff
@@ -0,0 +1,119 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+#import <EXUpdates/EXUpdatesAppController+Internal.h>
+#import <EXUpdates/EXUpdatesAppLauncherWithDatabase.h>
+#import <EXUpdates/EXUpdatesConfig.h>
+#import <EXUpdates/EXUpdatesDevLauncherController.h>
+#import <EXUpdates/EXUpdatesReaper.h>
+#import <EXUpdates/EXUpdatesReaperSelectionPolicyDevelopmentClient.h>
+#import <EXUpdates/EXUpdatesRemoteAppLoader.h>
+#import <EXUpdates/EXUpdatesSelectionPolicy.h>
+#import <EXUpdates/EXUpdatesUpdate.h>
+#import <React/RCTBridge.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+static NSString * const EXUpdatesDevLauncherControllerErrorDomain = @"EXUpdatesDevLauncherController";
+
+@implementation EXUpdatesDevLauncherController
+
+@synthesize bridge = _bridge;
+
++ (instancetype)sharedInstance
+{
+  static EXUpdatesDevLauncherController *theController;
```

Comment:
```suggestion
  __block static EXUpdatesDevLauncherController *theController;
```
Don't we need this if we are setting `theController` inside the ^{} block?

Reply:
`__block` is to override `auto`. I'm not sure `__block static` would compile since block storage and static storage are likely mutually exclusive; block storage needs to be associated with the closure, while static storage is global to the process.

Reply:
ah i see.